finstmt/findata/database.py

```diff
@@ -134,3 +127,39 @@ def get_sympy_subs_dict(self, t_offset: int = 0) -> Dict[IndexedBase, float]:
                 self.prior_statement.get_sympy_subs_dict(t_offset=t_offset - 1)
             )
         return subs_dict
+
+    #### NEW
+
+    t = symbols("t", cls=Idx)
+
+    # Get item even if attribute exists
+    def __getattribute__(self, key: str):
+        # print("FinDataBase.__getattribute__", key)
+
+        if key == "items_config":
+            return object.__getattribute__(self, key)
```

Comment:
Seems like this should be caught by the check directly below already.

Though it raises another issue. If we are allowing people to declare whatever config items they want, they could conflict with existing attribute names. We'll need to add checks such that they can only add non-reserved names. Would be a good item for a follow-up PR.

Reply:
all done now except for non-reserved name checking.